_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -1399,6 +1399,7 @@ bool MfxHwH264Encode::IsMctfSupported(
     mfxExtCodingOption2 const & extOpt2 = GetExtBufferRef(video);
     isSupported = (IsOn(extOpt2.ExtBRC) &&
         IsExtBrcSceneChangeSupported(video) &&
+        (video.mfx.FrameInfo.Width <= 3840 && video.vpp.In.Height <= 2160) &&
```

Comment:
Sebastian, could you say where it comes from?

Reply:
This is an actual limitation for MCTF, it can denoise frames up to 3840x2160.